build/NuGetAdditionalFiles/RunCsc

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Once again: why does that matter to me? If you're doing something unsupported why should I give you a friendly name to do that easier?